doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -96,38 +96,124 @@ to accessible objects.  (see Section :ref:`invocation-of-tactics`).
 
    .. seealso:: Section :ref:`performingcomputations`.
 
-.. cmd:: Search {+ {? - } @search_item } {? {| inside | outside } {+ @qualid } }
+.. cmd:: Search {+ @search_query } {? {| inside | outside } {+ @qualid } }
 
-   .. insertprodn search_item search_item
+   This command can be used to filter the goal and the global context
+   to quickly retrieve what one is looking for.  Library files that
```

Comment:
"quickly" is optimistic! There is no reason that it is always quick. It is dependent on the size of the context and on the complexity of the request.